test/functional/unified-spec-runner/entities.ts

```diff
@@ -116,6 +111,10 @@ export class UnifiedMongoClient extends MongoClient {
       ...(description.ignoreCommandMonitoringEvents ?? []),
       'configureFailPoint'
     ];
+    // FIXME: hack to get tests passing, extra unexpected events otherwise
```

Comment:
can we add a jira ticket for this?

Reply:
Yep I'm just first going to confirm this is still an issue with the LB work merged in, I know there were a number of changes made to the unified runner there and I put this hack in before LB was done.

Reply:
Confirmed it's still an issue and filed NODE-3549